### ISSUE TITLE
add Cyrus Yip's blog

### ISSUE THREAD

**Issue by CyrusYip** [sellorm/awesome-blogdown#80]
## New Site

**NAME:** Cyrus Yip's blog

**URL:** https://cyrusyip.org/

**Description:** Cyrus Yip's blog on daily life

**Comment by sellorm** [comment]
Hi Cyrus, is your site built with [blogdown](https://pkgs.rstudio.com/blogdown/), or with something else?

**Comment by CyrusYip** [comment]
> Hi Cyrus, is your site built with [blogdown](https://pkgs.rstudio.com/blogdown/), or with something else?

Yes, it is built with blogdown using [hugo-theme-even](https://github.com/olOwOlo/hugo-theme-even). Source code is [here](https://github.com/CyrusYip/cyrusyip-blog).

**Comment by sellorm** [comment]
Thanks, it's now live on the site - sorry this took me so long to get to! 😬
